sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -848,3 +848,36 @@ CREATE TABLE `benefit_eligibility` (
 #IfTable eligibility_response
 DROP TABLE `eligibility_response`;
 #Endif
+
+#IfTable erx_weno_drugs
+CREATE TABLE IF NOT EXISTS `erx_weno_drugs` (
+  `drug_id` int(11) NOT NULL AUTO_INCREMENT,
+  `rxcui_drug_coded` int(11) DEFAULT NULL,
+  `generic_rxcui` int(11) DEFAULT NULL,
+  `drug_db_code_qualifier` text,
+  `full_name` varchar(250) NOT NULL,
+  `rxn_dose_form` text,
+  `full_generic_name` varchar(250) NOT NULL,
+  `brand_name` varchar(250) NOT NULL,
+  `display_name` varchar(250) NOT NULL,
+  `route` text,
+  `new_dose_form` varchar(100) DEFAULT NULL,
+  `strength` varchar(15) DEFAULT NULL,
+  `supress_for` text,
+  `display_name_synonym` text,
+  `is_retired` text,
+  `sxdg_rxcui` varchar(10) DEFAULT NULL,
+  `sxdg_tty` text,
+  `sxdg_name` varchar(100) DEFAULT NULL,
+  `psn-drugdescription` varchar(100) DEFAULT NULL,
+  `ncpdp-quantity-term` text,
+  `potency-unit-code` varchar(10) DEFAULT NULL,
+  `dea-schedule-no` int(2) DEFAULT NULL,
+  `dea-schedule` varchar(7) DEFAULT NULL,
+  `ingredients` varchar(100) DEFAULT NULL,
+  `drug-interaction` varchar(100) DEFAULT NULL,
+  `unit-source-code` varchar(3) DEFAULT NULL,
+  `code-list-qualifier` int(3) DEFAULT NULL,
+  PRIMARY KEY (`drug_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=22017;
```

Comment:
Why is the AUTO_INCREMENT set to 22017?

Reply:
I left it at this number after I did the import. If anyone goes in and adds to this table. I thought it would be good to leave this number set. I know my ignorance is showing. But was not sure if it was not set would MYSQL automatically set it when the import is done. I think I just answered my own question. I will remove it.

Reply:
Still need to remove this AUTO_INCREMENT